ios/Mappers.swift

```diff
@@ -144,6 +144,10 @@ class Mappers {
         if let offlineDetails = paymentIntent.offlineDetails {
             offlineDetailsMap = mapFromOfflineDetails(offlineDetails)
         }
+        var metadataMap: NSDictionary?
+        if let refundMetadata = paymentIntent.metadata {
+            metadataMap = NSDictionary(dictionary: refundMetadata)
+        }
```

Comment:
nit

```suggestion
        var metadataMap: NSDictionary?
        if let intentMetadata = paymentIntent.metadata {
            metadataMap = NSDictionary(dictionary: intentMetadata)
        }
```

Reply:
oops fixed in https://github.com/stripe/stripe-terminal-react-native/pull/656/commits/0af8410c38c94f08bd17c7c97a108ef16bd1f1d0